spec/02-integration/18-hybrid_rpc/05-notification_spec.lua

```diff
@@ -0,0 +1,123 @@
+local helpers = require "spec.helpers"
+
+local function test_url(path, port, code, headers)
+  helpers.wait_until(function()
+    local proxy_client = helpers.http_client("127.0.0.1", port)
+
+    local res = proxy_client:send({
+      method  = "GET",
+      path    = path,
+      headers = headers,
+    })
+
+    local status = res and res.status
+    proxy_client:close()
+    if status == code then
+      return true
+    end
+  end, 10)
+end
+
+-- register a test rpc service in custom plugin rpc-notification-test
+for _, strategy in helpers.each_strategy() do
+  describe("Hybrid Mode RPC #" .. strategy, function()
+
+    lazy_setup(function()
+      helpers.get_db_utils(strategy, {
+        "clustering_data_planes",
+      }) -- runs migrations
```

Comment:
This could be the reason for flakiness. Please clean all the tables, to ensure the `test_url("/001", 9002, 200)` does not confuse with previously configured routes

Reply:
Could you show me how to do? thanks.